src/redux/configureStore.js

```diff
@@ -0,0 +1,12 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import roomsReducer from './rooms/rooms';
+
+const reducer = combineReducers({
+  roomsReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger, thunk));
+
```

Comment:
Please kindly include the `compose` which handles nested functions during API calls. It looks like this:
```
import {
  createStore, combineReducers, applyMiddleware, compose,
} from 'redux';

const store = createStore(
  reducer,
  compose(applyMiddleware(thunk, logger)),
);
```